indico/legacy/webinterface/wcomponents.py

```diff
@@ -380,7 +381,8 @@ def getVars( self ):
                     selected = "selected"
             else:
                 selectedSession = "all"
-            sessions = [ i18nformat(""" <select name="showSession" onChange="document.forms[0].submit();" style="font-size:8pt;"><option value="all" %s>- -  _("all sessions") - -</option> """)%selected]
+            sessions = ["""<select name="showSession" onChange="document.forms[0].submit();" style="font-size:8pt;">
```

Comment:
yeah, just noticed it. maybe not worth fixing now since this will eventually be rewritten anyway